src/app/caution/entry-notice.tsx

```diff
@@ -0,0 +1,30 @@
+import { Text } from "@/components/ui/text";
+import Section from "./section";
+import styles from "./section.module.css";
+
+const EntryNotice: React.FC = () => {
+  return (
+    <Section title="入退場について" id="entry-notice">
```

Comment:
idに特に意図が無いのであれば削除してください
```suggestion
    <Section title="入退場について">
```